.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+<!--
+Thanks for taking the time to contribute to DVT!
+
+Please ensure that your pull request title matches the conventional commits
+specification: https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/CONTRIBUTING.md#conventional-commits
+-->
+
+## Description of changes
+_Write a description of the changes you have made in this PR. Extremely small changes such as fixing typos do not need a description._
+
+## Issues to be closed
+_Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google._
+
+Closes #ISSUE_NUMBER
+
+<!--
+For example, if your pull requests resolves issues 1000 and 2000 write:
+* Closes #1000
+* Closes #2000
+-->
+
+## Checklist
+- [ ] I have followed the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/CONTRIBUTING.md).
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have successfully run the E2E tests, and have included the outcomes in the PR comments
```

Comment:
I feel like some people (especially external contributors) use PRs as a way of running the tests.. so maybe that shouldn't be a checklist item.